doc/ENVIRONMENT.md

```diff
@@ -96,3 +96,19 @@ systemd-timedated:
   NTP client services. If set, `timedatectl set-ntp on` enables and starts the
   first existing unit listed in the environment variable, and
   `timedatectl set-ntp off` disables and stops all listed units.
+
+systemd itself:
+
+* `$SYSTEMD_ACTIVATION_UNIT` — set for all NSS and PAM module invocatins that
```

Comment:
s/invocatins/invocations/
Also, do you want to codify here that the value is the name of the parent unit? (I'm not sure if it is implicit here or intentionally left undefined)

Reply:
hmm, what do you mean by "parent" unit? it's the unit that we run the NSS/PAM lookups for.

Reply:
I was only asking if you want to specify in this doc what the value for the environment flag is supposed to be (as the modules may want checking for specific values, like this PR does).